node/src/container_chain_spawner.rs

```diff
@@ -410,7 +504,7 @@ impl ContainerChainSpawner {
             // Edge case: when starting the node it may be assigned to a container chain, so we need to
             // start a container chain already collating.
             let start_collation = Some(para_id) == current;
-            self.spawn(para_id, start_collation).await;
+            self.spawn(para_id, start_collation, false).await;
```

Comment:
Do we need warp sync to be enabled in this case?

Reply:
No, because that could potentially delete the db. So the first call to spawn will be warp_sync = false, and the recursive call will be warp_sync = true.